public/app/features/explore/TraceView/TraceView.test.tsx

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { TraceView } from './TraceView';
+import { SpanData, TraceData, TracePageHeader, TraceTimelineViewer } from '@jaegertracing/jaeger-ui-components';
+
+function renderTraceView() {
+  const wrapper = shallow(<TraceView trace={response} />);
+  return {
+    timeline: wrapper.find(TraceTimelineViewer),
+    header: wrapper.find(TracePageHeader),
+    wrapper,
+  };
+}
+
+describe('TraceView', () => {
```

Comment:
So most of the hooks are also tested through the testing of the component itself. Not sure if we have any example of hook tests already and whether we generally test hooks separately. In this case I assume they won't be reused so thought this was fine atm. @dprokop @hugohaggmark

Reply:
Sorry I can't recall creating any hook test, but I think @dprokop created tests for the LogRowContext?

Reply:
There is a hook test example for useLokiLabels